### DISCUSSION TITLE
Link Between Custom Fields

### DISCUSSION THREAD

**Discussion by remchuk** [netbox-community/netbox#17854] [Help Wanted!]
Is it possible?
I want to create a Custom field on the Device and call it, Secondary_IP.

I want that when adding a new IP Address and setting true  to a custom field called Is_secondary_ip it will show it on the device  secondary IP.

Is it a possibility in the Netbox?

**Comment by betadrome** [comment]
> Is it possible? I want to create a Custom field on the Device and call it, Secondary_IP.
> 
> I want that when adding a new IP Address and setting true to a custom field called Is_secondary_ip it will show it on the device secondary IP.
> 
> Is it a possibility in the Netbox?

If I have understood correctly, you want to decide when the secondary IP address is displayed in the device! There is no way to decide when which data is displayed if this data exists (you can only set the complete custom field data as "hidden". However, it is possible to configure the custom field so that it is only displayed in the device if it contains data. This function is called "hidden if unset"

**Reply by remchuk** [reply]
Not quite what I want.

I want to connect to custom fields.

When setting custom field A and assigning it to a device, a custom field B which is on the device, will be set.


It work like primary Ip 
When creating new ip and setting is_primary to true and assiging it to a device, it will appear on the device under the Primary IP
